_doc/Spec_Copy.md

```diff
@@ -2,9 +2,70 @@
 
 The copy system allows an existing creature to be copied, with modifications, to create a new creature.
 
-## Usage
+## Homebrew Metadata
 
-```yaml
+When using `"_copy"` as part of a homebrew file, dependencies need to be declared so that the site loads data in the correct order.
+The `"_meta"` property can contain a list of `"dependencies"`, `"internalCopies"` and `"includes"` when needed.
+
+### dependencies
+
+Dependencies are declared on site source files as a map in the following format:
+
+```jsonc
+"<data property>": [
+    "<JsonSource1>",
+    ...,
+    "<JsonSourceN>"
+]
+```
+
+#### Note
+
+Entities from these sources can then be extended/referenced in this file. When copying classes/subclasses/class features/subclass features, the array should consist of `"<classIdentifier1>", ..., "<classIdentifierN>"` items, where `"classIdentifierN"` matches the keys in `"5etools/data/class/index.json"`.
+
+When copying class/etc. homebrew, normal "source"-based linking applies.
+
+### internalCopies
+
+An array of keys that are copied from within the current document. e.g. `"item"`, `"monsterFluff"`, `"background"` etc.
+
+### includes
+
+`"includes"` is structured similarly to `"dependencies"` and enables the site to load additional content from other homebrew files that are stored in the homebrew repo.
+
+### Example
+
+An example homebrew file might contain the following information in it's `_meta` property:
+
+```jsonc
+"dependencies": {
+    "monster": [    //Use the property name first
```

Comment:
```suggestion
    "monster": [    // Use the property name first
```

spaces after yer `//`s

Reply:
Yessir will fix tomorrow when PC is on 👍